### ISSUE TITLE
[Feature request] more time-to-now choices in alert query

### ISSUE THREAD

**Issue by rtbellows** [grafana/grafana#7777]
Would be nice to get more time ranges included in the alert eval query, currently it looks like:

1s, 10s, 1m, 5m, 10m, 15m, 1h, 24h, 48h

for selectable values. It would be great to break down that large jump from 1h to 24h in 1h or 2h increments.

Secondly, it would be fantastic to be able to specify an alert notification only after X amount of alert occurrences. Thanks!

**Comment by jkv** [comment]
+1 to "specify an alert notification only after X amount of alert occurrences"

**Comment by ashokrajar** [comment]
IMO, notification only after X amount of alert occurrences is a must to have.

**Comment by fredsmithjr** [comment]
Since we don't have: "specify an alert notification only after X amount of alert occurrences"
Currently have to do a gross hack job of 
if avg() of query A from 5m,now AND
avg() of query A from 10m,now-5m AND
avg() of query A from 15m,now-10m AND
avg() of query A from 20m,now-15m 

In order to begin to replicate successive failures.

This would really make the grafana alerts more likely to replace our current system

**Comment by tubby1981** [comment]
Need this too

**Comment by Krinkle** [comment]
I'd suggest to narrow down this feature request to just be about "More time-to-now choices".

The separate feature request to be able to make an alert only fire after matching a certain number of times (or after a certain amount of time), is already tracked at https://github.com/grafana/grafana/issues/7320.
